R/az_daily.R

```diff
@@ -86,6 +86,7 @@ az_daily <- function(station_id = NULL, start_date = NULL, end_date = NULL) {
     )) %>%
     dplyr::filter(.data$meta_station_id != "az99") %>%
     dplyr::mutate(datetime = lubridate::ymd(.data$datetime)) %>%
+    dplyr::mutate(wind_2min_timestamp = lubridate::with_tz(lubridate::parse_date_time(.data$wind_2min_timestamp, orders = "ymdHMSz"), tzone = "America/Phoenix")) %>%
     #convert NAs
     dplyr::mutate(
       dplyr::across(
```

Comment:
Just below this is code to change values like -9999 to NA, but it currently skips `wind_2min_timestamp`.  I think all you need to do is:

```r
...
dplyrr::across(
  c(wind_2min_timestamp, tidyselect(where(is.numeric))), 
  function(x)
  ...
```

Reply:
Without adding the above code to include `wind_2min_timestamp` in the conversion of NAs, existing code with `wind_2min_timestamp` as character automatically converts corresponding `nodata` values of `-99999` to `NA` while throwing a warning `27 failed to parse`, as with

`az_daily(start_date = "2023-03-22", end_date = "2023-03-22")`

Only one station had `wind_2min_timestamp` daily values on that date.

I think the `nodata` value conversion needs to happen somehow in conjunction with character-to-datetime conversion of this variable.

Reply:
Also, the above code has trouble with `wind_2min_timestamp`, as it is an object of class "c('POSIXct', 'POSIXt')"

Reply:
@Aariq please review solutions in `az_daily()` and `az_hourly()` I just pushed on this branch. I think I figured it out